components/Search.tsx

```diff
@@ -7,10 +7,17 @@ import FormationCard from "@/components/FormationCard";
 import { ScrollArea } from "@/components/ui/scroll-area";
 import { Separator } from "@/components/ui/separator";
 import { Label } from "@/components/ui/label";
+import { FormationData } from "@/lib/formations";
 
-export default function Search({ cmsData }: { cmsData: any }) {
+export default function Search({
+  cmsData,
+  prePopulated,
+}: {
+  cmsData: any;
```

Comment:
I know this wasn't changed this PR, but not sure this should be `any` - atleast `Record<string, any` would be better. I assume the cmsData follows some kind of strict format?

Reply:
Yeah it does, mostly me being lazy here and not writing the type signature out lol. I can do that